tests/epyccel/test_epyccel_functions.py

```diff
@@ -448,6 +448,19 @@ def f(a : T, b : T):
     with pytest.raises(TypeError):
         epyc_f(3.5, 4)
 
+@pytest.mark.parametrize( 'language', (
+        pytest.param("fortran", marks = pytest.mark.fortran),
+        pytest.param("c", marks = pytest.mark.c),
+    )
+)
```

Comment:
In Python we do not have type checks because these are generated by the Python wrapper?